code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1184,10 +1203,10 @@
 
 	var/armor_block = H.run_armor_check(affecting, "melee", "<span class='notice'>Your armor has protected your [hit_area].</span>", "<span class='notice'>Your armor has softened a hit to your [hit_area].</span>",I.armour_penetration)
 	var/Iforce = I.force //to avoid runtimes on the forcesay checks at the bottom. Some items might delete themselves if you drop them. (stunning yourself, ninja swords)
-
-	var/dmgcheck = apply_damage(I.force, I.damtype, affecting, armor_block, H)
-
-	if(!dmgcheck && I.force != 0 || !affecting) //Something went wrong. Maybe the limb is missing?
+	var/dmgcheck = apply_damage((1-I.stamina_percentage)*I.force, I.damtype, affecting, armor_block, H)
+	var/fakedmgcheck = apply_damage(I.stamina_percentage*I.force, STAMINA, affecting, armor_block, H)
```

Comment:
should give a better name to the var to understand what it does, i am a bit confused here